pom.xml

```diff
@@ -131,10 +131,17 @@
             <version>2.13</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>ssh-credentials</artifactId>
+          <version>1.13</version>
+          <scope>test</scope>
+          <classifier>tests</classifier>
```

Comment:
PCT error indicated that ssh-credentials:1.11 is getting in the way. pipeline-model-definition has transitive dependency on 1.11 thru credentials-bindings plugin. I think we should try without `tests` classifier as I suspect plugin resolution might still fail. https://gist.github.com/vivek/a8f26d392ceb461a608da01ad2159f82

Reply:
It might do it?